code/game/machinery/cloning.dm

```diff
@@ -170,7 +169,7 @@
 			occupant.adjustCloneLoss(-2 * heal_rate)
 
 			//Premature clones may have brain damage.
-			occupant.adjustBrainLoss(-1 * heal_rate)
+			occupant.adjustBrainLoss(-(n_ceil(0.5*heal_rate)))

```

Comment:
this should not be `n_ceil`; for one, that's a useless proc, for another it's incorrect (it calculates `floor(x) + 1`, not `ceil(x)`; these are different for integer values)

`#define ceil(x) (-round(-(x)))` somewhere in helpers, and use `ceil()`.